src/lib/charts/BarChart/roundedBarCharts.ts

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
Don't examine this *too* closely. It was copied almost verbatim from an article/gist I found.